src/components/group/ohri-obs-group.component.tsx

```diff
@@ -40,12 +41,18 @@ export const OHRIObsGroup: React.FC<ObsGroupProps> = ({ question, onChange, dele
         return (
           <div className={`${styles.flexColumn} ${styles.obsGroupColumn} `}>
             {isUnspecifiedSupported(field) ? (
-              <>
+              <div className={styles.parent}>
                 {questionFragment}
-                <OHRIUnspecified question={field} onChange={onChange} handler={formFieldHandlers[field.type]} />
-              </>
+                <div className={styles.tooltipWithUnspecified}>
+                  <OHRIUnspecified question={field} onChange={onChange} handler={formFieldHandlers[field.type]} />
+                  {field.questionInfo && <OHRITooltip field={field} />}
+                </div>
+              </div>
             ) : (
-              questionFragment
+              <div className={styles.parent}>
+                {questionFragment}
+                <div className={styles.tooltip}>{field.questionInfo && <OHRITooltip field={field} />}</div>
+              </div>
```

Comment:
Having 2 different references looks unnecessary. This can be condensed by renaming the class to some thing more generic and then adding the check only to show the Tooltip component